adm/project.go

```diff
@@ -238,6 +238,9 @@ func GetCurrentDirectory() string {
 }
 
 func installProjectTmpl(p Project, cfg *config.Config, cfgFile string, info *dbInfo) {
+	if p.Driver == "oceanbase" {
+		p.Driver = "OceanBase"
```

Comment:
Need comment for this replace. And maybe it's better to keep the Driver as `oceanbase` to avoid mistakes to be introduced by others in the future.